libs/moment-adapter/ng-package.json

```diff
@@ -0,0 +1,15 @@
+{
+    "$schema": "../../node_modules/ng-packagr/ng-package.schema.json",
+    "dest": "../../dist/libs/moment-adaptepter",
```

Comment:
is this path correct? `moment-adaptepter` ?

Reply:
yep
![image](https://user-images.githubusercontent.com/5276786/126775011-9de3682a-5f68-4e1c-8d0d-5f62d5a33e48.png)

Reply:
@xandrio `moment-adaptepter ` != `moment-adapter`